src/test/java/de/tum/in/www1/artemis/authentication/JiraAuthenticationIntegrationTest.java

```diff
@@ -165,9 +160,7 @@ void testJWTAuthentication() throws Exception {
         HttpHeaders httpHeaders = new HttpHeaders();
         httpHeaders.add("User-Agent", "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36");
 
-        UserJWTController.JWTToken jwtToken = request.postWithResponseBody("/api/authenticate", loginVM, UserJWTController.JWTToken.class, HttpStatus.OK, httpHeaders);
-        assertThat(jwtToken.getIdToken()).as("JWT token is present").isNotNull();
-        assertThat(this.tokenProvider.validateTokenForAuthority(jwtToken.getIdToken())).as("JWT token is valid").isTrue();
+        request.postWithoutResponseBody("/api/authenticate", loginVM, HttpStatus.OK, httpHeaders);
```

Comment:
Same here, should we check for a correctly set cookie? If yes, please adapt similar occurrences

Reply:
Done in [65dbec5](https://github.com/ls1intum/Artemis/pull/5723/commits/65dbec5a53bd612cc76ec8d287bc963fc92517ec)!